api/api.go

```diff
@@ -262,9 +262,21 @@ func configureFSSOpenam(fasit *FasitClient, request *NamedConfigurationRequest,
 		return appErr
 	}
 
-	appErr = fasit.PostFasitResource(payload, request)
+	resourceRequest := createResourceRequest(payload.Alias, payload.ResourceType)
```

Comment:
Maybe just send a pointer to payload and get the values from the payload object inside createResourceRequest?